cernopendata/modules/fixtures/data/records/cms-validation-code-Castor2010.json

```diff
@@ -60,7 +60,7 @@
       ]
     },
     "usage": {
-      "description": " <p>Run this software in the CMS Open Data container or in the the CERN Virtual Machine for 2010 CMS data installed. For the container, follow the instructions for 2010 data in <a href=\"docs/cms-guide-docker\">Running CMS analysis code using Docker</a>. For the virtual machine, follow the instructions in step 1 at <a href=\"/VM/CMS/2010\">How to install a CERN Virtual Machine</a>.</p> "
+      "description": " <p>Run this software in the CMS Open Data container or in the the CERN Virtual Machine for 2010 CMS data installed. For the container, follow the instructions for 2010 data in <a href=\"http://opendata.cern.ch/docs/cms-guide-docker\">Running CMS analysis code using Docker</a>. For the virtual machine, follow the instructions in step 1 at <a href=\"/VM/CMS/2010\">How to install a CERN Virtual Machine</a>.</p> "
```

Comment:
We can use here `<a href=\"/docs/...` so that the link would work also with HTTPS. I'll amend this during merge.

Reply:
👍 

It was a missing `/` in the link then. As it was it brought to `.../records/docs/...` which did not exist

Reply:
Yes, exactly :+1:  The link was considered relative, not absolute, due to missing of leading `/`.